src/Symfony/Bundle/FrameworkBundle/Test/ContainerAwareTestCase.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Test;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * This class can be used as a base class for functional tests that need to
+ * access the service container.
+ *
+ * @author Matthias Pigulla <mp@webfactory.de>
+ */
+class ContainerAwareTestCase extends KernelTestCase
```

Comment:
@xabbuh it is abstract so that PHPUnit does not detect it as a TestCase needing to run by itself.